toolchain/binutils/Makefile

```diff
@@ -71,6 +71,11 @@ ifneq ($(CONFIG_EXTRA_TARGET_ARCH),)
 		--enable-targets=$(call qstrip,$(CONFIG_EXTRA_TARGET_ARCH_NAME))-linux-$(TARGET_SUFFIX)
 endif
 
+ifneq ($(CONFIG_GOLD_SUPPORT),)
+  HOST_CONFIGURE_ARGS+= \
+		--enable-gold
```

Comment:
I think this can be enabled unconditionally

Reply:
Okay, I will modify my PR for just enabling gold linker support…

Reply:
done